VENTokenField/VENTokenField.h

```diff
@@ -40,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (NSUInteger)numberOfTokensInTokenField:(VENTokenField *)tokenField;
 - (NSString *)tokenFieldCollapsedText:(VENTokenField *)tokenField;
 - (UIColor *)tokenField:(VENTokenField *)tokenField colorSchemeForTokenAtIndex:(NSUInteger)index;
+- (Class)tokenViewForTokenField:(VENTokenField *)tokenField;
```

Comment:
okay, I think I get your logic, so instead of supplying a custom view here, you are just supplying the class name and then the caller of the data source can actually instantiate the view?